lib/SparkPost/Transmission.php

```diff
@@ -63,12 +63,22 @@ private function __construct() {}
 	 */
 	private static function getHttpClient() {
 		if(!isset(self::$request)) {
-			self::$request = new Client();
+			$hostConfig = SparkPost::getConfig();
+			$defaultOptions = [
+				'base_uri'=>self::getBaseUrl($hostConfig),
+				'headers' =>[
+					'Authorization' => $hostConfig['key'],
+					'Content-Type' => 'application/json',
+					'User-Agent' => 'SparkPost PHP SDK v0.1.2'
```

Comment:
@aydrian just pointed out that we use this format in the node SDK: `'User-Agent': 'node-sparkpost/' + this.version`.  We should probably follow a similar format.